### DISCUSSION TITLE
ubuntu-18-04%E6%90%AD%E5%BB%BA%E9%82%AE%E7%AE%B1%E7%AC%94%E8%AE%B0postfixdovecotmysql/

### DISCUSSION THREAD

**Discussion by giscus** [wl223600/Dovecote#5] [Discussion]
# ubuntu-18-04%E6%90%AD%E5%BB%BA%E9%82%AE%E7%AE%B1%E7%AC%94%E8%AE%B0postfixdovecotmysql/

由于服务器数据迁移，论坛注册时发送验证码用的邮箱配置废掉了。趁着五一假期有空，先在目前这台服务器上把他搞上。 为了安全起见及日后扩展需求，这次采用了虚拟用户的方案，也就是借助SQL来实现多域、多用户。和以往一样，这次也是使用了TLS。 搭建过程主要参考了这位老哥的文章：https://www.cnblogs.com/renweihang/p/7988591.html，总体质量不错，适用于Ubunt

https://fatshibainu.me/ubuntu-18-04%E6%90%AD%E5%BB%BA%E9%82%AE%E7%AE%B1%E7%AC%94%E8%AE%B0postfixdovecotmysql/

**Comment by wl223600** [comment]
🤣
